kuma/wiki/models.py

```diff
@@ -811,6 +809,10 @@ def revert(self, revision, user, comment=None):
         revision.created = datetime.now()
         revision.creator = user
         revision.save()
+        # TODO: change to primary key check instead of object comparison
```

Comment:
Maybe. But we're already changing so much with the update branch that I'm trying to minimize changes for now.